From 0227a4229179f32aa20fdab400e5c47cc8c9cfb6 Mon Sep 17 00:00:00 2001
From: Ben Webb <ben@salilab.org>
Date: Fri, 11 Oct 2024 21:04:35 -0700
Subject: [PATCH] Replace deprecated PyEval_CallObject

PyEval_CallObject is deprecated and is
removed in Python 3.13. Replace with PyObject_Call.
---
 pyext/mdt_callbacks.i | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyext/mdt_callbacks.i b/pyext/mdt_callbacks.i
index 9409db3b18a0971180df2d8f547d5c204b33f7eb..6c29946fa60986af8a00d488a28fc9c669e71318 100644
--- a/pyext/mdt_callbacks.i
+++ b/pyext/mdt_callbacks.i
@@ -16,7 +16,7 @@ static float *python_cb_get_property(gpointer data,
     return NULL;
   }
 
-  result = PyEval_CallObject(func, arglist);
+  result = PyObject_Call(func, arglist, NULL);
   Py_DECREF(arglist);
 
   if (result) {
